src/components/ContentSidebar/DetailsSidebar.js

```diff
@@ -54,7 +52,7 @@ type State = {
     classification?: ClassificationInfo,
     classificationError?: Errors,
     isLoadingClassification: boolean,
-    file: BoxItem,
+    file?: BoxItem,
```

Comment:
Is this still used? If not, let's remove it.

Reply:
This is the state, which is still used to store the result of the files call (cached)

Reply:
That's my mistake, I thought I was looking in Props.